### DISCUSSION TITLE
Unable to Resume After Pausing?

### DISCUSSION THREAD

**Discussion by ShannonVanshoon** [AndreyPavlenko/Fermata#539] [General]
I've been using Fermata Auto for a while now, and I've been enjoying it... but I keep running into a lingering issue: a lot of the time, I'm able to play a video in Android Auto, but after pausing it for whatever reason, then attempting to play, it just gets 'stuck', and nothing happens. I can hit pause again, then play, but it still just stays there. Attempting to play any other video doesn't work, and I have to close out of the program entirely.

I thought I fixed this issue by setting the movie engine to 'MediaPlayer' instead of the default 'VLC', which worked for a while, but then it suddenly stopped working altogether. I also tried clearing all the data from the application and trying it again, but still no dice. This is on a Samsung Galaxy A71 5G running Android 13, for reference. I've also attached the very latest set of logfiles for review to this reply. Can anyone confirm what's going on?

[Fermata (1).log](https://github.com/user-attachments/files/17237358/Fermata.1.log)

**Comment by AndreyPavlenko** [comment]
MediaPlayer fails to play this file with MEDIA_ERROR_UNKNOWN error and it's switched to VLC.

**Comment by ShannonVanshoon** [comment]
Well, that'd definitely explain it, yes... huh. Has this come up before, and is there any potential way to fix this? I've been really enjoying the program, to be honest, so I'd like to correct this if I can.

Thanks for the quick reply as well!

**Reply by AndreyPavlenko** [reply]
Try disabling VLC in settings.

**Comment by ShannonVanshoon** [comment]
... And I *completely* forgot that that was an option, yes, so apparently I'm an idiot. Alright, thanks; looks like I'm showing some positive results so far, so if anything comes up, I'll let you know. Thanks so much again!

**Reply by ShannonVanshoon** [reply]
Alright, disabling VLC definitely fixed the issue with resuming after pausing... though I am curious: do you know why the program'd try to use the VLC engine instead of MediaPlayer, as selected? I don't at all mind using MediaPlayer, but uninstalling the VLC engine removes the file-system option for it as well, and without it, Fermata tends to struggle with pulling up data on video files.

**Reply by AndreyPavlenko** [reply]
The app uses the next engine in the list when it failed to play. VLC is default for video, because it has more features and codecs. VLC is also used to retrieve the metadata, but if it's disabled, the built into Android metadata retriever is used.
